sys-schema.md

```diff
@@ -0,0 +1,35 @@
+---
+title: sys Schema
+summary: 了解 sys 库中的系统表。
+---
+
+# `sys` Schema
+
+> **注意：**
+>
+> 从 v8.0.0 及之前版本升级的集群将不包含 `sys` Schema 和其中的 View。它们可以通过以下 SQL 语句创建：
+>
+> {{< copyable "sql" >}}
+>
+> ```sql
+> CREATE DATABASE IF NOT EXISTS sys;
+> CREATE OR REPLACE VIEW sys.schema_unused_indexes AS
+>   SELECT
+>     table_schema as object_schema,
+>     table_name as object_name,
+>     index_name
+>   FROM information_schema.cluster_tidb_index_usage
+>   WHERE
+>     table_schema not in ('sys', 'mysql', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA') and
+>     index_name != 'PRIMARY'
+>   GROUP BY table_schema, table_name, index_name
+>   HAVING
+>     sum(last_access_time) is null;
+> ```
```

Comment:
````suggestion
TiDB 从 v8.0.0 开始提供 `sys` Schema。你可以通过查看 `sys` 系统数据库中的表或视图理解 TiDB 的系统表、[`INFORMATION_SCHEMA` 表](/information-schema/information-schema.md)和 [`PERFORMANCE SCHEMA` 表](/performance-schema/performance-schema.md)内的数据。

对于从 v8.0.0 之前版本升级的集群，`sys` Schema 和其中的视图不会自动创建。你可以通过以下 SQL 语句手动创建：

```sql
CREATE DATABASE IF NOT EXISTS sys;
CREATE OR REPLACE VIEW sys.schema_unused_indexes AS
  SELECT
    table_schema as object_schema,
    table_name as object_name,
    index_name
  FROM information_schema.cluster_tidb_index_usage
  WHERE
    table_schema not in ('sys', 'mysql', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA') and
    index_name != 'PRIMARY'
  GROUP BY table_schema, table_name, index_name
  HAVING
    sum(last_access_time) is null;
```
````